kernel/fs/iso9660.inc

```diff
@@ -736,3 +736,1044 @@ cd_compare_name:
         pop     esi eax edi edx
         stc
         ret
+
+;;; New driver
```

Comment:
doesn't need, it was already said in the commit info

Reply:
this comment is for further code modification as a delimiter of the old and new driver. In one of the following patches, it will be necessary to remove the old code and this comment will point to the new driver

Reply:
> this comment is for further code modification as a delimiter of the old and new driver. In one of the following patches, it will be necessary to remove the old code and this comment will point to the new driver

you could say it in comment. Letters don't cost money